src/control/server/storage/bdev/runner.go

```diff
@@ -90,50 +90,48 @@ func defaultScriptRunner(log logging.Logger) *spdkSetupScript {
 	}
 }
 
-// Reset executes setup script to deallocate hugepages & return PCI devices
-// to previous driver bindings.
-//
-// NOTE: will make the controller reappear in /dev.
-func (s *spdkSetupScript) Reset() error {
-	out, err := s.runCmd(s.log, nil, s.scriptPath, "reset")
-	return errors.Wrapf(err, "spdk reset failed (%s)", out)
-}
```

Comment:
I think I understand the motivation behind removing this in favor of using `req.ResetOnly` -- it's to take advantage of common code, right? It does lead to some awkwardness on the part of the caller and reader, though. Seems like it would be better to keep this `Reset()` method and just refactor the common code out into a helper method called by this one and `Prepare()`.

Reply:
done